scripts/check_known_pulsars.py

```diff
@@ -231,6 +232,38 @@ def find_pulsars_power(obsid, powers=None, names_ra_dec=None):
 
     return pulsar_power_dict, meta_data
 
+def get_sources_in_fov(obsid, source_type, fwhm):
```

Comment:
doesn't this do the same thing as the new script you made?

Reply:
This just does the basics. My new script actually calls this ~6 times

Reply:
Shouldn't this function be in the new script? this is a large part of what it does and seems like an odd choice to put it in here

Reply:
Check_known_pulsars.py already did a similar thing I just made it into a function to neaten up the code and reduce redundancies